src/briefcase/platforms/android/gradle.py

```diff
@@ -170,6 +170,26 @@ def output_format_template_context(self, app: AppConfig):
             build = int(getattr(app, "build", "0"))
             version_code = f"{v[0]:d}{v[1]:02d}{v[2]:02d}{build:02d}".lstrip("0")
 
+        # The default runtime libraries included in an app. The default value is the
+        # list that was hard-coded in the Briefcase 0.3.16 Android template, prior to
+        # the introduction of customizable system requirements for Android.
+        try:
+            dependencies = app.system_runtime_requires
+        except AttributeError:
+            self.logger.warning(
+                (
+                    "The Android configuration for this app does not contain "
+                    "a `dependencies` definition. Using a default value; you should "
+                    "add an explicit list of Android dependencies for the project."
```

Comment:
Most users will have no clue how to do that, so the message should link to the documentation page with the default list.